Merge to core repo clone #122

dh1542 opened the pull request.

Timm638 committed:
Adds Notebook for Normalization

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Extends notebook by Duplicate Detection

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Adds K Sigma Filtering as Test

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Finished up plot of notebook

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge branch 'develop' into feature/043_rtdip_demo

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Extends notebook & fixes bug in Value Imputation

- Refactored notebook
- Adds ARIMA & Value Imputation test
- Fixes missing check in value imputation

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Adds notes to the notebook

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Applies black to code

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Started to refactor Arima

- Adds Input Validator
- Refactors helper methods out of arima.py

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
refactors arima.py

- Update workaround to take from project
- Removed unnecessary variable assignment

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge branch 'develop' into feature/040_finishing_ARIMA

FelipeTrost committed:
feat: schema validation for anomaly detection

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

Timm638 committed:
refactors arima.py for preparation of exogenous variables

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
refactors AutoArima to its own Component

Signed-off-by: Timm638 <Timm638@gmail.com>

FelipeTrost committed:
Merge remote-tracking branch 'origin/develop' into refactor/63-anomal-detection-tests

Timm638 committed:
adds auto-recognition of column names to ARIMA

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
cleans up RegEx

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
tries to implement exog

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
splits up Arima into two files

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
prepare test data in test_arima.py

Signed-off-by: Timm638 <Timm638@gmail.com>

FelipeTrost committed:
Merge remote-tracking branch 'origin/develop' into refactor/63-anomal-detection-tests

FelipeTrost committed:
test: ksigma anomaly detection, wrong values and large data set

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

chris-1187 committed:
refactorings on #61 and #92

Signed-off-by: chris-1187 <christian.munz@posteo.de>

chris-1187 committed:
actions adjustment

Signed-off-by: chris-1187 <christian.munz@posteo.de>

chris-1187 committed:
#65 ARIMA refactorings

Signed-off-by: chris-1187 <christian.munz@posteo.de>

Timm638 committed:
cleans up test

Signed-off-by: Timm638 <Timm638@gmail.com>

dh1542 committed:
Merge pull request #102 from amosproj/refactor/#92-MVI_#61-DD

refactorings on #61,#65 and #92

Commit:
#66 add datatype validation and prediction on large dataset

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
validate datatype

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
refactor invalid datatype

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

dh1542 committed:
Merge pull request #103 from amosproj/refactor/066_linear_regression_test

refactor/066_linear_regression

Timm638 committed:
documents code & finishes up column-conversion

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge branch 'develop' into feature/040_finishing_ARIMA

Timm638 committed:
adapted test cases to new ARIMA structure

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
applies Black linter

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
applies workaround to test dataframes

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
fixes import

Signed-off-by: Timm638 <Timm638@gmail.com>

dh1542 committed:
Merge pull request #94 from amosproj/feature/040_finishing_ARIMA

#40: Finishing Arima

dh1542 committed:
Merge pull request #101 from amosproj/refactor/63-anomaly-detection-tests

refactor/63 anomaly detection tests

dh1542 committed:
Updated gitignore after merge

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
Merge remote-tracking branch 'origin/develop' into develop

dh1542 committed:
Added deliverables for sprint-09

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

FelipeTrost committed:
refactor(normalization): use input validator

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
refactor(test/normalization): add tolerance for data frame comparison

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
test(normalization): test idempotence with large data set

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
test(normalization): test wrong type

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

Timm638 committed:
Merge branch 'develop' into feature/096_rtdip_demo_pipeline

# Conflicts:
#	src/sdk/python/rtdip_sdk/pipelines/data_quality/data_manipulation/interfaces.py

Timm638 committed:
fixes imports by re-adding them to each data quality component

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
fixes imports by re-adding them to each data quality component

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
fixes imports by re-adding them to each data quality component

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
fixes imports by re-adding them to each data quality component

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge remote-tracking branch 'origin/feature/096_rtdip_demo_pipeline' into feature/096_rtdip_demo_pipeline

# Conflicts:
#	src/sdk/python/rtdip_sdk/pipelines/data_quality/monitoring/spark/__init__.py

Timm638 committed:
renamed imports

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge remote-tracking branch 'origin/feature/115_fixing_missing_import' into feature/115_fixing_missing_import

# Conflicts:
#	src/sdk/python/rtdip_sdk/pipelines/data_quality/monitoring/spark/__init__.py

Timm638 committed:
Applies black to files

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Applies black to files

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
fixes imports & writes down pipeline concept

Signed-off-by: Timm638 <Timm638@gmail.com>

dh1542 committed:
Merge pull request #114 from amosproj/refactor/64-de-normalization-tests

refactor/64 de normalization tests

mollle committed:
added sprint 10 deliverables folder

Signed-off-by: mollle <56913823+mollle@users.noreply.github.com>

mollle committed:
added sprint 10 deliverables

Signed-off-by: mollle <56913823+mollle@users.noreply.github.com>

mollle committed:
remove unnecessary file

mollle committed:
delete out of value ranges done #109

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
Merge pull request #116 from amosproj/feature/115_fixing_missing_import

Fix Imports for Data Quality Pipeline Components

mollle committed:
Merge pull request #116 from amosproj/feature/115_fixing_missing_import

Fix Imports for Data Quality Pipeline Components

mollle committed:
imporved documentation

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
flaltline filter done

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
added documentation for my components

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
Merge branch 'develop' into feature/109_delete_out_of_range_values

mollle committed:
Merge pull request #119 from amosproj/feature/109_delete_out_of_range_values

delete out of value ranges done #109

mollle committed:
Merge pull request #120 from amosproj/feature/113_remove_flatlining_rows

Feature/113 remove flatlining rows

mollle committed:
fixed documentation

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
fix value range test

Signed-off-by: mollle <leonmariusmoll@gmail.com>

FelipeTrost committed:
docs: added docs entries for amos components

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
fix: return type annotations in linear_regression

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

Timm638 committed:
updates showcase_notebook to current version of rtdip

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
starts working on pipeline_showcase.ipynb

Signed-off-by: Timm638 <Timm638@gmail.com>

FelipeTrost committed:
fix: type annotations for older python versions in linear_regression

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

chris-1187 committed:
Added deliverables for sprint-11

Signed-off-by: chris-1187 <christian.munz@posteo.de>

mollle committed:
Merge pull request #121 from amosproj/110-docs

docs: added docs entries for amos components

mollle committed:
renamed DeleteOutOfRangeValues to OutOfRangeValueFilter

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
fixed tests after renaming OutOfRangeValueFilter

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
107 moving average done

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
fixing typo in docs file

Signed-off-by: mollle <leonmariusmoll@gmail.com>

dh1542 committed:
#106: Added files

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
#106: Implemented gaussian smoothing wip

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
Merge remote-tracking branch 'origin/develop' into merge/to-core

dh1542 committed:
Merged amos develop to merge branch

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 commented:
@Amber-Rigg @TugceOzberkYener We have another set of changes. We are awaiting your feedback. Thanks in advance!

mollle committed:
fixed build docu

Signed-off-by: mollle <leonmariusmoll@gmail.com>

FelipeTrost committed:
docs: add nav entries for amos components

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
Merge remote-tracking branch 'origin/develop' into 110-docs

FelipeTrost committed:
docs: correct reference to value filter

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
docstring: fixes for arima

- removed inline comments because giffe (parser for docs) was throwing warnings
- removed parameter from docstring which the function didn't have

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
docs: add arima

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

TugceOzberkYener commented:
Thank you for the mkdocs documentation web page updates.
Regarding with the changes there are 2 things which needs attention before we merge the pull request:
1 - There are markdown files which are listed in the docs directory but not included in the nav configuration. Let's add them in the nav at appropriate location.
<img width="902" alt="image" src="https://github.com/user-attachments/assets/044e9ed6-cec9-43b7-8e38-e4632904df49" />

2 - There is a reference to 'sdk/code-reference/pipelines/data_quality/data_manipulation/spark/delete_out_of_range_values.md' in the 'nav' configuration, which is not found in the documentation files. Which causes 404 page not found error at the web page:
<img width="1501" alt="image" src="https://github.com/user-attachments/assets/9deaae24-bcb5-4c52-9533-a9d1b4d0fe1e" />

Timm638 committed:
saves in progress pipeline_showcase changes

Signed-off-by: Timm638 <Timm638@gmail.com>

mollle committed:
Merge pull request #127 from amosproj/110-docs

110 docs

mollle committed:
test if htm files are readable in github

Signed-off-by: mollle <leonmariusmoll@gmail.com>

TugceOzberkYener commented:
Currently the new components has been added to the following paths;
pipelines.data_quality.data_manipulation --> contains components that returns the new data frame after applying requested changes
pipelines.data_quality.monitoring --> contains components that does not manipulate the input data frame - only logs the anomalies
pipelines.machine_learning --> currently only contains the linear regression
 
Can you please rename the "machine_learning" package into "forecasts" and move the ARIMA Prediction class into this package next to linear regression. We will then have 2 components which can predict/forecast.

TugceOzberkYener reviewed (changes requested):
Added 2 comments regarding with the mkdocs documentation and package renaming for component locations. Could you please apply the requested changes.

Timm638 committed:
adds Sensor Data Stage

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
fixes copy bug in ARIMA

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
push latest pipeline changes

Signed-off-by: Timm638 <Timm638@gmail.com>

chris-1187 committed:
RTDIP Blog Post

Signed-off-by: chris-1187 <christian.munz@posteo.de>

chris-1187 committed:
Agile illustration

Signed-off-by: chris-1187 <christian.munz@posteo.de>

Timm638 committed:
completes draft of notebook

Signed-off-by: Timm638 <Timm638@gmail.com>

luccalb commented:
@TugceOzberkYener thank you for your feedback!
I've created two followup Issues to take care of your comments #128 and #129 which we will plan for this sprint.

mollle committed:
Merge branch 'develop' into feature/107_MovingAverage

Signed-off-by: mollle <56913823+mollle@users.noreply.github.com>

dh1542 committed:
Merge pull request #125 from amosproj/feature/107_MovingAverage

107 moving average done

dh1542 committed:
Merge branch 'refs/heads/develop' into feature/#106-Gaussian-Smoothing

dh1542 committed:
#106: Implementation and refactoring gaussian smoothing

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
#106: Added correct documentation for gaussian smoothing

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
#106: Linted

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

Timm638 committed:
changed dataset presented to be a more periodic one

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
cleaned up draft more, each cell except for Linear Regression functions

Signed-off-by: Timm638 <Timm638@gmail.com>

Commit:
Add deliverables for sprint-12

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
Merge branch 'develop' of https://github.com/amosproj/amos2024ws01-rtdip-data-quality-checker into develop

Timm638 committed:
adjusts plot to be more descriptive

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
adds Example Data to git

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
finishes up texts

Signed-off-by: Timm638 <Timm638@gmail.com>

mollle committed:
improving/fixing documentation

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
fixed/improved docu, renamed "machine_learning" into "forecasting"
Co-authored-by: Felipe Trost <felipe.trost@gmail.com>

Signed-off-by: mollle <leonmariusmoll@gmail.com>

mollle committed:
fixing tests that broke because of refactoring

mollle committed:
fixed arima test

Signed-off-by: mollle <leonmariusmoll@gmail.com>

Commit:
#105 refactor for only euclidean distance

Commit:
#105 add parameters for time series prediction

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
#105 add unit test for KNN

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
refactor

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
linted

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
debug import error

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
#105 Rename machine_learning to forecasting folder

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

kristen149 committed:
Merge branch 'develop' into feature/105_KNN

Signed-off-by: Minh Khue Tran <100759917+kristen149@users.noreply.github.com>

GBBBAS committed:
Refactor Time Series Builder Queries for Raw, Resample and Interpolate (#859)

* Refactor Time Series Builder Queries for Raw, Resample and Interpolate

Signed-off-by: GBBBAS <bryce.bartmann@shell.com>

* Add tests for 3.5.2

Signed-off-by: GBBBAS <bryce.bartmann@shell.com>

---------

Signed-off-by: GBBBAS <bryce.bartmann@shell.com>

GBBBAS committed:
Update tests (#860)

Signed-off-by: GBBBAS <bryce.bartmann@shell.com>

kristen149 committed:
Delete tests/sdk/python/rtdip_sdk/pipelines/forecasting/__init__.py

Signed-off-by: Minh Khue Tran <100759917+kristen149@users.noreply.github.com>

kristen149 committed:
Delete tests/sdk/python/rtdip_sdk/pipelines/forecasting/spark/__init__.py

Signed-off-by: Minh Khue Tran <100759917+kristen149@users.noreply.github.com>

FelipeTrost committed:
fix imports for builds

After a lot of pain, I figured out, that imports using src at the
beginning where being kept after the build step and made the built sdk
fail when importing one of those components

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

kristen149 committed:
Linted formatting

Signed-off-by: Minh Khue Tran <100759917+kristen149@users.noreply.github.com>

Timm638 committed:
updates nbformat

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge branch 'develop' into feature/096_rtdip_demo_pipeline

# Conflicts:
#	src/sdk/python/rtdip_sdk/pipelines/data_quality/data_manipulation/spark/__init__.py
#	src/sdk/python/rtdip_sdk/pipelines/data_quality/data_manipulation/spark/prediction/__init__.py
#	src/sdk/python/rtdip_sdk/pipelines/data_quality/monitoring/spark/__init__.py
#	src/sdk/python/rtdip_sdk/pipelines/forecasting/__init__.py

Timm638 committed:
fixed imports

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
removes redundant .gitignore
Signed-off-by: Timm638 <Timm638@gmail.com>

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
applies black

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
fixes missing preparation from pandas DF to spark DF

Signed-off-by: Timm638 <Timm638@gmail.com>

FelipeTrost committed:
fix more imports for build

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
initial notebook for shell demo

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

Timm638 committed:
reapplies black

Signed-off-by: Timm638 <Timm638@gmail.com>

mollle committed:
Merge pull request #136 from amosproj/feature/105_KNN

Feature/105_K Nearest Neighbors

chris-1187 committed:
basic md blog post files

Signed-off-by: chris-1187 <christian.munz@posteo.de>

dh1542 committed:
Merge branch 'develop' into merge/to-core

dh1542 committed:
#126: Merged new changes from develop

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 commented:
@TugceOzberkYener We implemented the feedback. Can you take a look at it?

dh1542 committed:
Merge remote-tracking branch 'upstream/develop' into merge/to-core

dh1542 committed:
Merge remote-tracking branch 'origin/develop' into feature/#106-Gaussian-Smoothing

dh1542 committed:
#106: Removed converting dfs in tests

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
#106: Linted

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
Merge remote-tracking branch 'upstream/develop' into develop

# Conflicts:
#	tests/api/v1/test_api_batch.py
#	tests/api/v1/test_api_circular_average.py
#	tests/api/v1/test_api_circular_standard_deviation.py
#	tests/api/v1/test_api_interpolate.py
#	tests/api/v1/test_api_interpolation_at_time.py
#	tests/api/v1/test_api_latest.py
#	tests/api/v1/test_api_metadata.py
#	tests/api/v1/test_api_plot.py
#	tests/api/v1/test_api_raw.py
#	tests/api/v1/test_api_resample.py
#	tests/api/v1/test_api_sql.py
#	tests/api/v1/test_api_summary.py
#	tests/api/v1/test_api_time_weighted_average.py
#	tests/api/v1/test_api_utilities.py

dh1542 committed:
#106: Changed implementation without converting to panda df

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
#106: Fixed static method

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 committed:
#106: Fixed import

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

luccalb committed:
feat: demo day slide

kristen149 committed:
Merge pull request #134 from amosproj/feature/#106-Gaussian-Smoothing

#106 Gaussian Smoothing

Timm638 committed:
Adds demo video

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
updates imports to new ones
Signed-off-by: Timm638 <Timm638@gmail.com>

Signed-off-by: Timm638 <Timm638@gmail.com>

luccalb committed:
feat: update demo day slide

FelipeTrost committed:
Merge remote-tracking branch 'origin/develop' into demo-notebook

FelipeTrost committed:
sprint 13 deliverables

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
Merge pull request #138 from amosproj/sprint-13-amos-deliverables

sprint 13 deliverables

Timm638 committed:
Merge branch 'develop' into feature/096_rtdip_demo_pipeline

Timm638 committed:
adds variation of notebook with outputs

Signed-off-by: Timm638 <Timm638@gmail.com>

FelipeTrost committed:
Merge remote-tracking branch 'origin/develop' into demo-notebook

FelipeTrost committed:
fix: print correct column name in error

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
use relative imports to fix build

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
final demo notebook

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
fix typo

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

TugceOzberkYener commented on src/sdk/python/rtdip_sdk/queries/time_series/time_series_query_builder.py:
```diff
@@ -225,6 +227,7 @@ def raw(
             "end_date": end_date,
             "include_bad_data": include_bad_data,
             "display_uom": display_uom,
+            "sirt": sort,
```
small typo "sirt" should be "sort"

TugceOzberkYener commented on src/sdk/python/rtdip_sdk/queries/time_series/time_series_query_builder.py:
```diff
@@ -316,8 +323,10 @@ def resample(
             "time_interval_rate": time_interval_rate,
             "time_interval_unit": time_interval_unit,
             "agg_method": agg_method,
+            ":fill": fill,
```
typo ":fill" should be "fill"

mollle committed:
docu except guassian smoothing complete

Signed-off-by: mollle <leonmariusmoll@gmail.com>

FelipeTrost committed:
format

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

FelipeTrost committed:
user documentation

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

Timm638 committed:
Merge remote-tracking branch 'origin/demo-notebook' into feature/096_rtdip_demo_pipeline

# Conflicts:
#	src/sdk/python/rtdip_sdk/pipelines/forecasting/spark/data_binning.py

chris-1187 committed:
image adjustments

Signed-off-by: chris-1187 <christian.munz@posteo.de>

Timm638 committed:
merges changes from Felipe's Notebook to this branch

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
changes something to trick GitHub into showing the pull request as successful.

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge pull request #139 from amosproj/demo-notebook

Imports notebook with import fixes to notebook branch

Timm638 committed:
Pulls changes from notebook branch without the notebooks

Signed-off-by: Timm638 <Timm638@gmail.com>

mollle committed:
Merge pull request #142 from amosproj/feature/141-import-bugfixes-changes-from-notebook-to-develop

Feature/141 import bugfixes changes from notebook to develop

dh1542 committed:
Added missing documentation for gaussian smoothing

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

chris-1187 committed:
Added code and charts

Signed-off-by: chris-1187 <christian.munz@posteo.de>

mollle committed:
fixed docu build warning

Signed-off-by: mollle <leonmariusmoll@gmail.com>

FelipeTrost committed:
Merge remote-tracking branch 'origin/develop' into 124/documentation

FelipeTrost committed:
deliverables docs: gaussian smoothing

Signed-off-by: Felipe Trost <felipe.trost@gmail.com>

mollle committed:
Merge pull request #140 from amosproj/124/documentation

user documentation

mollle committed:
fixing spelling issues, closes #145

dh1542 committed:
Merge branch 'develop' into merge/to-core

# Conflicts:
#	tests/api/v1/test_api_batch.py
#	tests/api/v1/test_api_circular_average.py
#	tests/api/v1/test_api_circular_standard_deviation.py
#	tests/api/v1/test_api_interpolate.py
#	tests/api/v1/test_api_interpolation_at_time.py
#	tests/api/v1/test_api_latest.py
#	tests/api/v1/test_api_metadata.py
#	tests/api/v1/test_api_plot.py
#	tests/api/v1/test_api_raw.py
#	tests/api/v1/test_api_resample.py
#	tests/api/v1/test_api_sql.py
#	tests/api/v1/test_api_summary.py
#	tests/api/v1/test_api_time_weighted_average.py
#	tests/api/v1/test_api_utilities.py

dh1542 committed:
Fixed spelling mistakes

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>

dh1542 commented:
@TugceOzberkYener We fixed the spelling in the queries.

dh1542 committed:
Merge pull request #143 from amosproj/documentation/123_Article_RTDIP_AMOS

Documentation/123 article rtdip amos

Timm638 committed:
Upload sprint 14 deliverables
Signed-off-by: Timm638 <Timm638@gmail.com>

Signed-off-by: Timm638 <Timm638@gmail.com>

Timm638 committed:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
edit example documentation for KNN

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

Commit:
edit example documentation for KNN

Signed-off-by: Minh Khue Tran <minh.khue.tran@fau.de>

dh1542 committed:
Merge branch 'develop' into merge/to-core

dh1542 committed:
Merge changes from develop

Signed-off-by: Dominik Hoffmann <dominik151099@outlook.de>
